content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -129,18 +134,13 @@ needed, because Docker 18.09 ships with containerd and both are detectable.
 If any other two or more runtimes are detected, kubeadm will exit with an appropriate
 error message.
 
-On non-Linux nodes the container runtime used by default is Docker.
-
-If the container runtime of choice is Docker, it is used through the built-in
-`dockershim` CRI implementation inside of the `kubelet`.
-
-Other CRI-based runtimes include:
+On non-Linux nodes, kubeadm uses Docker as the default container runtime.
 
-- [containerd](https://github.com/containerd/cri) (CRI plugin built into containerd)
-- [cri-o](https://cri-o.io/)
-- [frakti](https://github.com/kubernetes/frakti)
+If you select Docker as your container runtime, or use that by default, the kubelet
+integrates with Docker through the built-in `dockershim` CRI implementation.
```

Comment:
Lines 139-140, Could these lines be simplified to:
The kubelet integrates with Docker through the built-in `dockershim` CRI implementation.

Reply:
nit: Does this statement apply to Linux + other OSes:
By default, Kubernetes uses the Container Runtime Interface (CRI) to interface with your chosen container runtime.
nit: I read through the content using the tab UI. This line is duplicated:
If you select Docker as your container runtime, or use that by default, the kubelet integrates with Docker through Docker’s built-in dockershim CRI implementation.

Reply:
>  The kubelet integrates with Docker through the built-in dockershim CRI implementation.

Nice.

That duplication between tabs is intentional.

Reply:
I'm not well up on non-Linux container runtimes, so these changes will need technical review.

Reply:
dockershim applies to all OSes.
once/if dockershim goes outside of the kubelet, this text will need an update.
until then this is fine.